slsim/Sources/SourceVariability/variability.py

```diff
@@ -64,3 +119,130 @@ def variability_at_time(self, observation_times):
         :return: variability at given time.
         """
         return self._model(observation_times)
+
+    def parse_kwargs_for_lamppost_reprocessed_model(self):
```

Comment:
This function is still inside the Variability class. Is it possible to move this function outside of variability class? If it is necessary to put this function inside the variability class, I am ok with this.